test/tests/deformation/nek_standalone/nek.i

```diff
@@ -85,7 +85,7 @@
     value_type = min
   []
   [min_p_output]
-    type = NodalExtremeValue
+    type = ElementExtremeValue
```

Comment:
Why this change?

Reply:
This seems to work with both boundary and volume mirrors (we had a separate discussion about this on Slack). Please let me know if you would like me to change it to something else.

Reply:
NodalExtremeValue should also work with both boundary and volume mirrors (the only difference is whether the extreme value is searched over the DOFs themselves (the node values) vs. by evaluating the shape functions (the elemental value)).

Reply:
That's correct, and I have changed it back to `NodalExtremeValue`. Thanks!